test/massive/munit/TestClassHelperTest.hx

```diff
@@ -35,32 +35,33 @@ class TestClassHelperTest
 	public function new() {}
 
 	@Test
-	public function testConstructor():Void
+	public function testConstructor()
 	{
 		var helper:TestClassHelper = new TestClassHelper(TestClassStub);
 		
 		Assert.isNotNull(helper.test);
 		Assert.isType(helper.test, TestClassStub);
 		Assert.areEqual(helper.type, TestClassStub);
-		
+		#if !php
```

Comment:
Is there a PHP target restriction?

Reply:
now this is a forced change, it will be fixed in the PR for HashLink